### PR TITLE
Update v1 widget

### PR DESCRIPTION
**Pull request by diondrapeck**
- add hyperdrive features
 - update design
 - add warnings

### CODE REVIEW

**Comment by diondrapeck** [comment]
hyperdrive_run
script_run

**Comment by ninghu** [comment]
Can we hide warning if no warning?

File: R/run.R [hunk 1]
```diff
@@ -534,48 +534,105 @@ log_table_to_run <- function(name, value, description = "", run = NULL) {
...
 
-#' Plot table of run details in RStudio Viewer
+#' Plot table of run details in RStudio Viewer or browser
+#' This table does not auto-refresh. To see current values,
+#' re-run the command or click the web view link to view more
+#' details in real time.
 #' @param run The `Run` object.
 #' @export
 #' @md
 view_run_details <- function(run) {
```
Comment (heemanshusuri) [inline R/run.R]: does this function return anything that should be mentioned here?

Comment (diondrapeck) [reply R/run.R]: no - no return object other than the plot itself in the viewer

**Pull request merged**